### ISSUE TITLE
About Lora checkpoint

### ISSUE THREAD

**Issue by GitHub-TXZ** [hitachinsk/SAMed#10]
In the training process , this code: parser.add_argument('--lora_ckpt', type=str, default='/home/txz/github_code/SAMed/output/sam/results/AISD_512_pretrain_vit_b_epo199_bs12_lr0.005/epoch_159.pth', help='Finetuned lora checkpoint').  Can I think it as the continue training based on exists fintuned  lora version? But I have a bad results.

**Comment by hitachinsk** [comment]
I did not adopt continue training during the SAM adaptation, but only load the SAM model. Please leave this choice None.

**Comment by GitHub-TXZ** [comment]
if args.lora_ckpt is not None:
        net.load_lora_parameters(args.lora_ckpt)
this is your code, your learnable parameter concludes lora, mask decoder. and default prompt embedding. so if i just load lora, its SAM parts parameter is not loaded, so has worse result.  right ?

**Comment by hitachinsk** [comment]
Yes, you are right.

**Comment by GitHub-TXZ** [comment]
ok, 栓Q

**Comment by GitHub-TXZ** [comment]
And，Maybe it bothers you a lot, I have 4 RTX 3090, and I set bs = 15, which means batch size is 60. and Max epoch=999. But ,when epoch =19, my dc loss from 0.1 - 0.2 level to 0.4-0.6 level, it just  rises suddenly, and  I can't understand.

**Comment by GitHub-TXZ** [comment]
i am looking forward to your reply.

**Comment by GitHub-TXZ** [comment]
ok, thanks a lot
